website/docs/r/storage_bucket.html.markdown

```diff
@@ -90,7 +90,7 @@ The `action` block supports:
 
 * `storage_class` - (Required if action type is `SetStorageClass`) The target [Storage Class](https://cloud.google.com/storage/docs/storage-classes) of objects affected by this Lifecycle Rule. Supported values include: `MULTI_REGIONAL`, `REGIONAL`, `NEARLINE`, `COLDLINE`.
 
-The `condition` block supports the following elements, and requires at least one to be defined:
+The `condition` block supports the following elements, and requires at least one to be defined (If you specify multiple conditions in a rule, an object has to match all of the conditions for the action to be taken):
```

Comment:
Mind remove the parentheses? We may as well make it part of the normal sentence flow.

Reply:
sure! updated, PTAL.